code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/vip_extraction.dm

```diff
@@ -44,19 +44,22 @@
 	var/antag_elligable = FALSE
 	switch(pickweight(list("centcom_official" = 4, "dictator" = 1, "greytide" = 3)))
 		if("centcom_official")
-			created_human.flavor_text = "You are centcom official on board a badly damaged station. Making your way back to the station to uncover the secrets you hold is \
-				your top priority as far as Nanotrasen is concerned, but just surviving 1 more day is all you can ask for."
+			created_human.flavor_text = "You are a CentCom official onboard a badly damaged station. Making your way back to Space Station 13 to uncover the secrets you hold is \
+				your top priority as far as Nanotrasen is concerned, but surviving just one more day is all you can ask for."
+			created_human.mind.store_memory(created_human.flavor_text)
 			created_human.equipOutfit(/datum/outfit/centcom_official_vip)
 			antag_elligable = TRUE
 		if("dictator")
-			created_human.flavor_text = "It has been months since your regime fell. Once a hero, now just someone wishing that they will see the next sunrise. You know those \
+			created_human.flavor_text = "It has been months since your regime fell. Once a hero, you're now just someone wishing that they will see the next sunrise. You know those \
 				Nanotrasen pigs are after you, and will stop at nothing to capture you. All you want at this point is to get out and survive, however it is likely you will never leave \
 				without being captured."
+			created_human.mind.store_memory(created_human.flavor_text)
 			created_human.equipOutfit(/datum/outfit/vip_dictator)
 			created_human.mind.add_antag_datum(/datum/antagonist/vip_dictator)
 		if("greytide")
-			created_human.flavor_text = "You are just a lonely assistant, on a lonely derelict station. You dream of going home, \
+			created_human.flavor_text = "You are just an assistant on a lonely derelict station. You dream of going home, \
 				but it would take another one of the miracles that kept you alive to get you home."
+			created_human.mind.store_memory(created_human.flavor_text)
```

Comment:
This happens in every switch slot and is therefore redundant. Place it after the switch.

Reply:
Done